bin/mango-notebook

```diff
@@ -81,13 +81,14 @@ fi
 
 export PYSPARK_DRIVER_PYTHON=jupyter
 export PYSPARK_DRIVER_PYTHON_OPTS="notebook $NOTEBOOK_ARGS"
+export GOOGLE_CLOUD_NIO_JAR="${SCRIPT_DIR}/bin/google-cloud-nio-0.22.0-alpha-shaded.jar"
 
 PY4J_ZIP="$(ls -1 "${SPARK_HOME}/python/lib" | grep py4j)"
 export PYTHONPATH=${SPARK_HOME}/python:${SPARK_HOME}/python/lib/${PY4J_ZIP}:${PYTHONPATH}
 
 ${PYSPARK} \
   --conf spark.serializer=org.apache.spark.serializer.KryoSerializer \
   --conf spark.kryo.registrator=org.bdgenomics.mango.serialization.MangoKryoRegistrator \
-  --jars ${MANGO_CLI_JAR} \
+  --jars ${MANGO_CLI_JAR},${GOOGLE_CLOUD_NIO_JAR} \
```

Comment:
can we just add this as a spark argument instead of modifying? I dont think this belongs here because users aren't always running on google

Reply:
You can just add it as a spark arg (./bin/mango-notebook --jars ./bin/mango-notebook --jars ${GOOGLENIOJAR} --
This means you don't have to modify the actual file. This flag can be called in https://github.com/bigdatagenomics/mango/pull/340/files.
You can access this through docker by mounting the jar.

Also, this will only work for notebook, not browser.

Reply:
actually, you should just be able to do /bin/mango-notebook --jars https://oss.sonatype.org/content/repositories/releases/com/google/cloud/google-cloud-nio/0.22.0-alpha/google-cloud-nio-0.22.0-alpha-shaded.jar -- 

It acquires a bit of latency on the first call because you are downloading it upon the first count/collect. You can do it either way though.

Reply:
The default fs doesn't seem to work with the spark --jars command.
`SPARK_ARGS='--master yarn --jars https://oss.sonatype.org/content/repositories/releases/com/google/cloud/google-cloud-nio/0.22.0-alpha/google-cloud-nio-0.22.0-alpha-shaded.jar'
...

Exception in thread "main" java.io.IOException: No FileSystem for scheme: https
`

Reply:
ok then just wget it and add it as --jars

Reply:
Yep, this was the workaround used in the other branch (uploaded to gfs to reference).